web/frontend/src/i18n/es/index.ts

```diff
@@ -0,0 +1,4 @@
+
+export default {
+    where_to_question: '¿adonde?'
```

Comment:
I am not a Spanish speaker so most likely this needs to be reviewed later by someone who is. But I needed a second language to test it works.